### DISCUSSION TITLE
Extra boards

### DISCUSSION THREAD

**Discussion by admiralroflknife** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#285] [General]
I got this project working great for me and I now find myself with 4 extra boards after ordering for jlbpcb. Just the bare pcb but I don't want to waste them so if anyone in the lower 48 needs a board is be happy to mail it your way. It should fit fine in a standard envelope so for a dollar I can send it your way. 

Figured in case someone else didn't want to want the 4 weeks I did from jlbpcb.

**Comment by lyonrt** [comment]
I'm interested.  How can I contact you?

**Reply by BIIINO** [reply]
I am also interested. Send me ur contact on write me directly bino@imafex.sk

Thank you in advance

**Comment by evanlivingston** [comment]
I'm Interested

**Comment by Remigiusz2020** [comment]
HI,

Do you have still one left?
